rackcity/api/serializers/it_model_serializers.py

```diff
@@ -1,8 +1,19 @@
 from rest_framework import serializers
 from rackcity.models import ITModel
+from rackcity.api.serializers.fields import RCIntegerField
 
 
 class ITModelSerializer(serializers.ModelSerializer):
+
+    # This is the field for the serializers
+    # It overrides appropriate methods to work with empty strings when you call validate
+    memory_gb = RCIntegerField(
+        allow_null=True,
+        max_value=2147483647,
+        min_value=0,
+        required=False
+    )
```

Comment:
Because we're using a `serializers.ModelSerializer`, it would normally automatically detect that the field on the ITModel model is a `models.PositiveIntegerField`, and the serializer should use a `serializer.IntegerField` and set the values that you see here.  Instead, we force it to use our own version that overrides some methods of `serializer.IntegerField`